src/pages/TCY/hooks/useTcyClaimSaversNotification.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ToastId } from '@chakra-ui/react'
+import { useToast } from '@chakra-ui/react'
+import { useEffect, useRef } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useNavigate } from 'react-router-dom'
+
+import { TcyClaimSaversNotification } from '@/components/Layout/Header/ActionCenter/components/Notifications/TcyClaimSaversNotification'
+import { useWallet } from '@/hooks/useWallet/useWallet'
+import { ActionStatus } from '@/state/slices/actionSlice/types'
+import { preferences } from '@/state/slices/preferencesSlice/preferencesSlice'
+import { selectTcyClaimActionsByWallet } from '@/state/slices/selectors'
+import { useAppDispatch, useAppSelector } from '@/state/store'
+
+export const useTcyClaimSaversNotification = () => {
+  const toast = useToast()
+  const navigate = useNavigate()
+  const translate = useTranslate()
+  const dispatch = useAppDispatch()
+
+  const hasSeenTcyClaimAlert = useAppSelector(preferences.selectors.selectHasSeenTcyClaimAlert)
+  const toastIdRef = useRef<ToastId | undefined>(undefined)
+
+  const tcyClaimActions = useAppSelector(selectTcyClaimActionsByWallet)
```

Comment:
So yeah this is already filtered by wallet - if we did this on the notification itself, then we can simply return null from it instead of a hook